python/neuroglancer/server.py

```diff
@@ -30,6 +30,11 @@
 
 import sockjs.tornado
 
+try:
+    # Newer versions of tornado do not have the asynchronous decorator
+    from sockjs.tornado.util import asynchronous
+except (ImportError, ModuleNotFoundError):
```

Comment:
ModuleNotFoundError is a subclass of ImportError, so there is no need to include it.  Additionally, it doesn't exist on Python2, which Neuroglancer still suports.